addons/mrp/static/src/components/bom_overview_control_panel/mrp_bom_overview_control_panel.xml

```diff
@@ -3,50 +3,57 @@
+                    <div class="o_list_buttons o_mrp_bom_report_buttons d-xl-flex gap-xl-1">
                         <t t-if="props.showVariants">
-                            <button t-on-click="() => this.props.print(true)" type="button" class="btn btn-primary ms-1">Print All Variants</button>
+                            <button t-on-click="() => this.props.print(true)" type="button" class="btn btn-secondary">Print All Variants</button>
                         </t>
-                        <button t-on-click="clickUnfold" type="button" class="btn btn-primary ms-1">Unfold</button>
+                        <button t-on-click="clickUnfold" type="button" class="btn btn-secondary">Unfold</button>
                     </div>
                 </div>
             </t>
 
             <t t-set-slot="layout-actions">
-            <div class="d-flex flex-column gap-1">
-                <form class="d-flex w-100 gap-2">
-                        <label class="visually-hidden" for="bom_quantity">Quantity</label>
-                        <div class="input-group col-xl-4">
-                            <div class="input-group-text">Quantity</div>
-                            <input id="bom_quantity" type="number" step="any" t-on-change="ev => this.updateQuantity(ev)" t-on-keypress="ev => this.onKeyPress(ev)" t-att-value="props.bomQuantity" min="1" class="o_input form-control text-end"/>
-                            <div t-if="props.showOptions.uom" t-esc="props.uomName" class="input-group-text">Quantity</div>
-                        </div>
-                    <t t-if="props.showVariants" >
-                        <div class="input-group-text">Variant:</div>
-                        <div class="col-xl-10 col-lg-9 col-sm-8">
-                            <Many2XAutocomplete
-                                value="props.data.name"
-                                getDomain.bind="getDomain"
-                                resModel="'product.product'"
-                                fieldString="props.data.name"
-                                activeActions="{}"
-                                update.bind="(ev) => this.props.changeVariant(ev[0]?.id)"
-                            />
+            <div class="d-flex gap-1 w-100">
+                <form class="d-flex flex-grow-1 gap-2 flex-column flex-md-row">
+                    <label class="visually-hidden" for="bom_quantity">Quantity</label>
+                    <div t-attf-class="input-group align-items-center">
+                        <div class="pe-2 fw-bold">Quantity</div>
+                        <input id="bom_quantity" type="number" step="any" t-on-change="ev => this.updateQuantity(ev)" t-on-keypress="ev => this.onKeyPress(ev)" t-att-value="props.bomQuantity" min="1" class="o_input form-control rounded-0"/>
+                        <div t-if="props.showOptions.uom" t-esc="props.uomName" class="d-flex align-items-center text-muted small">Quantity</div>
```

Comment:
Could you replace deprecated `t-esc` by `t-out`? 

For the rest, LGTM ! Thanks for this, it looks much better 👍